incubator/hnc/pkg/stats/metrics.go

```diff
@@ -122,10 +128,13 @@ func recordMetric(m counter, ms *ocstats.Int64Measure) {
 	ocstats.Record(context.Background(), ms.M(int64(m)))
 }
 
-// recordTagMetric inserts a tag to the context before recording a metric.
+// recordObjectMetric inserts a KeyGroupKind tag to the context before recording a metric.
```

Comment:
Maybe add "if the object tag is not suppressed"?

Reply:
Done, also passing in the `GroupKind` directly now.